apps/docs/src/components/Downloads/slideTemplates.ts

```
@@ -0,0 +1,316 @@
+
+/**
+ * Build a complete presentation from templates
+ */
+export function buildPresentation(
+  slides: Omit<Slide, "number">[],
+  metadata?: {
+    addAgenda?: boolean;
+    addThankYou?: boolean;
+  },
+): Slide[] {
+  let allSlides = [...slides];
```

ESLint

[error] 'allSlides' is never reassigned. Use 'const' instead.

(prefer-const)